utils/virtual_machine/aws_provider.py

```diff
@@ -78,7 +78,7 @@ def pulumi_start_program():
                 "\n \n \n ❌ ❌ ❌ Exception launching aws provision step remote command ❌ ❌ ❌ \n \n \n "
             )
             vm_logger(context.scenario.name, context.vm_name).exception(pulumi_command_exception)
-            context.scenario.provision_status = "failed"
+            context.scenario.is_started_with_errors = True
```

Comment:
this method is called from the scenario warmup. We are catching the exception and we don't raise it. Another choice might be raise the exception an catch here:[]( https://github.com/DataDog/system-tests/blob/f9c6ddf129bce9cbf6af3e2c13546c00614d0318/utils/_context/_scenarios/core.py#L132)